source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py

```diff
@@ -231,6 +237,92 @@ def image(
     return images.clone()
 
 
+class image_features(ManagerTermBase):
+    """Extracted image features with a frozen encoder from images of a specific datatype from the camera sensor.
+
+    Calls :meth:`image` to get the images, then performs inference. On initialization,
+    for a model zoo different from the default, define model_zoo_cfg: A dictionary with string keys and callable values.
+    Should include "model", (mapped to a callable with no arguments to return the model), "preprocess" (mapped to
+    a callable which consumes the images and returns the preprocessed images),
+    and "inference" (mapped to a callable that provided the model, and the preproccessed images, returns the features.)
```

Comment:
Please use complete English sentences. As an example I modified the first line for you.

```suggestion
    This term calls the :meth:`image` function to obtain the sensor images. It then computes features
    on the obtained images through pre-terained networks.
    
    On initialization, for a model zoo different from the default, define model_zoo_cfg: A dictionary with string keys and callable values.
    Should include "model", (mapped to a callable with no arguments to return the model), "preprocess" (mapped to
    a callable which consumes the images and returns the preprocessed images),
    and "inference" (mapped to a callable that provided the model, and the preproccessed images, returns the features.)
```